htp/htp_parsers.c

```diff
@@ -183,9 +183,13 @@ int htp_parse_authorization(htp_connp_t *connp) {
         // Digest authentication
         connp->in_tx->request_auth_type = HTP_AUTH_DIGEST;
         return htp_parse_authorization_digest(connp, auth_header);
+    } else if (bstr_begins_with_c_nocase(auth_header->value, "bearer")) {
+        // Bearer authentication
+        connp->in_tx->request_auth_type = HTP_AUTH_BEARER;
```

Comment:
I guess you need to define `HTP_AUTH_BEARER` in  htp/htp_core.h following `HTP_AUTH_DIGEST`

Then, it would also be nice to have a test :
- one file in test/files/*.t that is the content of the traffic like `test/files/41-auth-digest.t`
- one function in test/test_main.cpp that read the contents of this file and tests the result like `TEST_F(ConnectionParsing, AuthDigest)`